packages/docs/docs/agent/bulk-status.md

```diff
@@ -0,0 +1,199 @@
+---
+sidebar_position: 2
```

Comment:
Recommend order as follows:
- Push to Agent
- System Requirements
- Agent Status
- Agent Bulk Status
- Troubleshooting
- Manual Uninstall

![image](https://github.com/medplum/medplum/assets/2314326/2f08d8ff-77f9-4c92-b309-cee95efc20a8)